doc/apl-blockchain.properties

```diff
@@ -332,6 +332,8 @@ apl.eth.node.url=https://ropsten.infura.io/v3/fd015ebfa03c41c1a3d4a24734da99b4
 
 apl.eth.pax.contract.address=0xd4aecf650c2c9a0d4e2bf08fd10268ca1e199fcd
 
+apl.eth.smart.contract.address=0xd811b4E6164B889Ed15196c14A051EA3a8EDF943
```

Comment:
Maybe rename property to 'apl.eth.contract.address'. Could you add some description for these properties?